query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/SourceValidationUtils.java

```diff
@@ -41,8 +41,7 @@ public SourceValidationUtils(ServiceReader serviceReader,
             ServiceActions serviceActions) {
         sourceUtilCommons = new SourceUtilCommons(managedServiceActions,
                 serviceActions,
-                serviceReader,
-                configuratorFactory);
+                serviceReader);
```

Comment:
I think there's a bit of code clean up we should consider doing in this class while we're refactoring all these methods. I think we should rename `sourceNameExists` to `duplicateSourceNameExits` and also rename `validateSourceName` to `duplicateSourceNameExits` since they are doing identical operations. The javadoc for both should look similar with the exception of the pid. I also think we should rename the `hasSourceName` method to `findSourceNameMatch`, mark it as private and remove the javadoc on it since it is really an implementation detail.